src/Settings/PreferencesWindow.cpp

```diff
@@ -121,6 +121,8 @@ PreferencesWindow::PreferencesWindow(QWidget *parent) : QMainWindow(parent)
 
     addPage("Extensions/CF Tool", {"CF/Path"});
 
+    addPage("File Path/Testcases", {"Testcases Matching Rules"});
```

Comment:
This location could be better like "Testcases" -> "Testcase Load rules". "Testcases" -> "Save test cases" and other test cases related functions could be there. But this is not an issue and we need to restructure some other settings and it could be done even after a merge. 👍  Good work